alert_banner.module

```diff
@@ -36,8 +40,8 @@ function alert_banner_entity_bundle_field_info_alter(&$fields, EntityTypeInterfa
 /**
  * Implements hook_page_top().
  */
-function alert_banner_page_top(array &$page_top) {
-  $settings = \Drupal::config('alert_banner.settings');
+function alert_banner_page_top(array &$page_top): void {
+  $settings = \Drupal::configFactory()->get('alert_banner.settings');
```

Comment:
Good use of `\Drupal::configFactory()` - this is the correct D10 pattern for accessing configuration.